src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,192 @@
+
+## Steps before Installing Okteto
+
+Before installing Okteto, you will need to follow these steps:
+
+- Create the `okteto` namespace
+- Allow containers as `root` in the `okteto` namespace
+- Allow Okteto Build service privileged access
+- Allow Okteto Daemon service to access the host
+- Copy Openshift wildcard certificate to the `okteto` namespace
+- Get the OpenShift Ingress Controller IP
+
+### Create the `okteto` namespace
+
+Run the following command to create the `okteto` namespace:
+
+```bash
+kubectl create namespace okteto
+```
+
+### Allow containers as `root` in the `okteto` namespace
+
+By default, running `root` containers in OpenShift is not allowed. Today, most of the Okteto Control Plane runs as `root` (this is something we will fix soon). To allow `root` containers for the Okteto Control Plane components, run the following commands:
+
+```bash
+oc adm policy add-scc-to-user anyuid -z default -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-buildkit -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-registry -n okteto 
+```
+
+### Allow Okteto Build service privileged access
+
+By default, running privileged containers in OpenShift is not allowed. This restriction impacts the installation of the Okteto Build. To give privileged access to the Okteto Build service, run the following command:
+
+```bash
+ oc adm policy add-scc-to-user privileged -z okteto-buildkit -n okteto
+```
+
+### Allow the Okteto Daemon service to access the host
+
+By default, access to the host in OpenShift is not allowed. This restriction impacts the installation of the Okteto Daemon service. To give access to the host to the Okteto Daemon run the following command:
+
+```bash
+oc adm policy add-scc-to-user hostaccess -z okteto -n okteto
+```
+
+### Copy OpenShift wildcard certificate to the `okteto` namespace
+
+OpenShift provides the wildcard certificate for the wildcard domain `apps.<your-openshift-subdomain>` . We recommend reusing it when installing Okteto.
+
+To do that, copy the certificate to the `okteto` namespace (replace `<cluster-name>` with your cluster name):
+
+```yaml
+kubectl get secret <clustername>-primary-cert-bundle-secret -n openshift-ingress -o yaml | sed 's/namespace: openshift-ingress/namespace: okteto/' | kubectl apply -f -
+```
+
+### Get the OpenShift Ingress Controller IP
+
+We recommend installing Okteto behind the Openshift Ingress controller. To do this, remember the IP of your Openshift Ingress Controller:
+
+```bash
+kubectl get svc router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+We will refer to this value using `<ingress-controller-ip>`.
+
+:::warning
+Installing Okteto behing the OpenShift Ingress controller means the following features aren't available:
```

Comment:
If I'm unfamiliar with Okteto, this warning creates a lot of noise. User is doing the recommended thing, but then we are telling me they will be missing features.  Not sure how we can frame this better, but I think that this creates noise and confusion.

Reply:
removed